test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -26,7 +26,7 @@ Filter with dateCreated in subscriptions
 --SHELL-INIT--
 dbInit CB
 brokerStart CB
-accumulatorStart
+accumulatorStart --pretty-print
```

Comment:
Ok. Please, include a reference to this thread comment in the issue, for reference.